include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -20,8 +20,8 @@
 
 #pragma once
 
-#if(ENABLE_HDF5 != 1)
-#    error The activated radiation plugin (radiation.param) requires HDF5
+#if(ENABLE_OPENPMD != 1)
+#    error The activated radiation plugin (radiation.param) requires openPMD-api with a HDF5 backend
```

Comment:
Your error message doesn't match the check.

Reply:
Yes, the `HDF5` flag actually is only available if `libSplash` is available, thus it does not check for hdf5. 
Since it will fail on run time with a message that it can not write the hdf5 file, I think this is fine for now, what do youthink @sbastrakov and @psychocoderHPC? (Or do we have a way to check whether the hdf5 backend in openPMD-api is available?)

Reply:
There is `openPMD::getVariants()` at runtime and `openPMD_HAVE_HDF5` at compile time.